docs/user/using-an-OIDC-provider-with-pinniped.md

```diff
@@ -60,3 +79,33 @@ clusters:
 The [Kubeapps auth-proxy configuration](./using-an-OIDC-provider.md#deploying-an-auth-proxy-to-access-kubeapps) remains the same as for the standard OIDC setup so that Kubeapps knows to deploy the auth-proxy service configured to redirect to your OIDC provider.
 
 With those changes, Kubeapps is ready to send any request for a specific cluster via Pinniped so that the OIDC `id_token` can be exchanged for client certificates accepted by the Kubernetes API server.
+
+## Debugging auth failures when using OIDC
+
+For general OIDC issues, have a look a [this OIDC debugging guide](./using-an-OIDC-provider-with-pinniped.md#debugging-auth-failures-when-using-OIDC).
+
+### Pinniped not trusting your OIDC
+
+There are some scenarios (e.g., TMC) in which Pinniped is not being bundled with the usual CA certificates. As a result, common OIDC providers (e.g.,  Google, VMware CSP login, etc.) are not trusted by Pinniped and, as result, the authentication in Kubeapps will always fail with a 401 status code.
+
+You can work around this issue by setting `spec.tls.certificateAuthorityData` in the `JWTAuthenticator` to match with the TLS CA used by the OIDC discovery endpoint.
+
+For instance, if adding Google as an OIDC provider, you will have to check the CA of the [OIDC discovery endpoint](https://accounts.google.com/.well-known/openid-configuration):
+
+```bash
+curl --insecure -vvI https://accounts.google.com/.well-known/openid-configuration 2>&1 | grep issuer
+
+*  issuer: C=US; O=Google Trust Services; CN=GTS CA 1O1
+```
+
+So we need to manually add the base64 data of the certificate `Google Trust Services GTS CA 1O1`. Having a look at [this Google repository](https://pki.goog/repository/) we retrieve the `.pem` and encode the content in base64:
+
+```bash
+curl -s https://pki.goog/repo/certs/gts1o1.pem | base64
+
+LS0tLS1CRUdJTiBDRVJUSU...
+``` 
+
+Next, use this `LS0tLS1CRUdJTiBDRVJUSU...` value as the `spec.tls.certificateAuthorityData` in your `JWTAuthenticator`.
+
+Proceed analogously with other OIDC providers. Also, you can use your browser's view certificate option to get the CA certificate.
```

Comment:
I'd mention this above as the main method - it's certainly what I think most people would find easier as a generic way to find the certificate details?